hash/sega_beena_cart.xml

```diff
@@ -31,10 +84,103 @@ license:CC0
+			</dataarea>
+			<dataarea name="pages" size="0x2500000"> <!-- book pages -->
+				<rom name="0 - 0001.png" size="0x1f405f5" crc="edc4474e" sha1="1efe8aed1e39c7a8e37ea9bb4eb15c6eb0a2164b" />
+				<rom name="0 - 0002.png" size="0x1f7cfe2" crc="c1486187" sha1="99f760de274dc1def1997ae369dc5cb994630934" />
+				<rom name="0 - 0003.png" size="0x1ce9d44" crc="e9f02350" sha1="bc62ac5a118856393378f6c7ba1f37064dbccd83" />
+				<rom name="0 - 0004.png" size="0x1fc7930" crc="a66d0a3f" sha1="c6d274e3057752067773729fa48be167cc569016" />
+				<rom name="0 - 0005.png" size="0x1d28699" crc="7a2addfd" sha1="99adfe0a3f1abb853982cf7c7f8bc06ec9396672" />
+				<rom name="0 - 0006.png" size="0x1f3acfa" crc="d5e6d4f1" sha1="52f6914643b703e662f3c72a8e3af842fdc9ac2d" />
+				<rom name="0 - 0007.png" size="0x1c22530" crc="f181f7ff" sha1="adfa30fa2de7e3113d6450c2827486c33a20430e" />
+				<rom name="0 - 0008.png" size="0x1fc964e" crc="113420bb" sha1="8bc8a7f1a2660292e9373eeafdde46c6137d208b" />
+				<rom name="0 - 0009.png" size="0x1d0443f" crc="1105baea" sha1="f0869e6ac9165cc9c984519d74c16f2badb1ae06" />
+				<rom name="0 - 0010.png" size="0x204a0d4" crc="2bf18016" sha1="6c7a69cc57224fbb77a77045f64ba5de8f21ae97" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="cookinbe" supported="no">
+		<description>Cooking Beena: o-Ryouri Dekichatta!</description>
+		<year>2007</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="クッキングビーナ ～おりょうりできちゃった！～"/>
+		<part name="cart" interface="sega_beena_cart">
+			<dataarea name="rom" size="0x800000" width="32" endianness="big">
+				<rom loadflag="load16_word_swap" name="S-100029-1000 - Cooking Beena-o-Ryouri Dekichatta (Japan).bin" size="0x800000" crc="a3f58ab8" sha1="20b3dfe395f7c97a32c93b580040787b98fd6024" />
+			</dataarea>
+			<dataarea name="pages" size="0x2500000"> <!-- book pages -->
+				<rom name="0 - 0001.png" size="0x1f0d2bd" crc="4bee410c" sha1="75c527111df0e12a960ba02a7d6bb023232dd901" />
+				<rom name="0 - 0002.png" size="0x1ddee9f" crc="bdcf11cb" sha1="654df738a2eb6aec22202a8de9963d0283d80383" />
+				<rom name="0 - 0003.png" size="0x1c8258a" crc="1daf3316" sha1="d7ad47a27b0afc12cc50a5a66d7bb65b870de82f" />
+				<rom name="0 - 0004.png" size="0x1e1e700" crc="458298f3" sha1="b8a72d18eca5b4a603bcb1efcafb7388516aab20" />
+				<rom name="0 - 0005.png" size="0x1e643db" crc="68f58736" sha1="7517f5df1f644cac392150a9e1b1f5bc0aee2130" />
+				<rom name="0 - 0006.png" size="0x1da0ea5" crc="9c567543" sha1="94520e8c1569114ca2b2147f814058073d511df5" />
+				<rom name="0 - 0007.png" size="0x1d9813e" crc="c5bd8898" sha1="b4a717e41959715310c85fab48fc99d5fc88a6e1" />
+				<rom name="0 - 0008.png" size="0x1b3d110" crc="db5ff567" sha1="2d4fcb531a1495630ee4404fbd03e3b7e014fbc9" />
+			</dataarea>
+		</part>
+	</software>
+
 	<software name="frpc" supported="no">
 		<description>Issho ni Henshin Fresh Pretty Cure</description>
```

Comment:
Should probably change this to “PreCure” to match the other ones, unless it actually says “Pretty Cure” in the cover art.

Reply:
Yes, the cover says "Fresh Pretty Cure"

![imagen](https://user-images.githubusercontent.com/5889052/111073087-affa7e00-84dd-11eb-9bb6-763d6e2bbfde.png)